MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,643 @@
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.LabelText = "FirstName";
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Add or remove Label text for editor
+
+By default the label should be generated by property name. You can remove label text by using the DataFormDisplayAttribute or using the `GenerateDataFormItem` event.
+
+### Using attribute
+
+You can remove the label text for editor by setting [ShowLabel] as false in dataformdisplayOption attribute.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string MiddleName { get; set; }
+    
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Using events
+
+You can remove the label text for editor by using the [ShowLabel] `Link` property in [DataFormItem] `Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ShowLable = false;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Changing layout settings
+
+You can change the label width, label position and editor width. This is possible by using the `GenerateDataFormItem` event. In the event, change the layout by using the [DefaultLayoutOption] property in the `DataFormItem`.
+
+By default label position is left.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Description")
+            {
+                e.DataFormItem.DefaultLayoutSettings.LabelPosition = DataFormLabelPosition.Top;
+                e.DataFormItem.DefaultLayoutSettings.LabelWidth = 100;
+                e.DataFormItem.DefaultLayoutSettings.EditorWidth = 200;
+            }
+                
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Background colour
+
+You can change the background colour for the label and editor by using `GenereateDataFormItem` event.
+
+You can change the background colour for the label and editor by using the [Background] `Link` property in [DataFormItem] `Link` .
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.Background = Colors.Azure; 
```

Comment:
Background is Brsuh type so use like Brush.Azure

Reply:
Brush.Azure added